### PR TITLE
Fresh PR for pycbc_astro

### PR DESCRIPTION
**Pull request by vaibhavtewari**
Hi @tdent I think I messed up my heads and branches and what not. Apologies about that. I am making a pull request which is everything which has been reviewed and approved + your fg_mc comment. Somehow I lost the commit history and couldnt figure out how to get it back.

### CODE REVIEW

File: pycbc/population/rates_functions.py [hunk 1]
```diff
@@ -10,8 +10,9 @@
 import bisect
 from pycbc.conversions import mchirp_from_mass1_mass2
 
-def process_full_data(fname, rhomin, mass1, mass2, lo_tlt_mass, hi_tlt_mass):
-    """Read the zero and time-lagged triggers identified by BBH templates.
+def process_full_data(fname, rhomin, mass1, mass2, lo_mchirp, hi_mchirp):
+    """Read the zero and time-lagged triggers identified by templates in
```
Comment (tdent) [inline pycbc/population/rates_functions.py]: kill whitespace here

File: pycbc/population/rates_functions.py [hunk 2]
```diff
@@ -112,9 +111,9 @@ def log_rho_bg(trigs, bins, counts):
     '''
 
     trigs = np.atleast_1d(trigs)
-
+
```
Comment (tdent) [inline pycbc/population/rates_functions.py]: kill whitespace here (and apparently other places?)

File: pycbc/population/rates_functions.py [hunk 3]
```diff
@@ -112,9 +111,9 @@ def log_rho_bg(trigs, bins, counts):
     '''
 
     trigs = np.atleast_1d(trigs)
-
+    
     N = sum(counts)
-
+    
     assert np.all(trigs >= np.min(bins)), 'triggers can not besmaller than bin lower limit'
```
Comment (tdent) [inline pycbc/population/rates_functions.py]: clarify error message : eg 'trigger SNR values cannot all be below the lowest bin limit'

**Comment by vaibhavtewari** [comment]
All fixed Tom. Please merge this one if you are happy. I will go little deeper in fgmc function and change it accordingly to keep it mathematically robust. Also I will change 2 functions that sample and fit skew log normal data. But will do that in a new PR.

**Comment by ahnitz** [comment]
@tdent @vaibhavtewari This was approved. Poking. Why is it not merged?

**Comment by tdent** [comment]
Codacy.

**Comment by ahnitz** [comment]
@vaibhavtewari The changes look pretty minor, can you make them and then let's merge this.

**Comment by tdent** [comment]
@vaibhavtewari I realize that the changes here may be somewhat out of date, but we should make at least some advance on this since it was approved in May.  I have dealt with the whitespace and line length issues and the necessary setup.py changes.  If you don't object I will merge within the next few hours and then continue developing from there (if wanted).

**Comment by vaibhavtewari** [comment]
@tdent thanks for the edits and merging. The module was complete when I last worked on it. There have been some improvements due to O2 analysis in the multiple files. All of them have been reviewed. I will try to update pycbc_astro accordingly.

**Pull request merged**